src/main/resources/application.properties

```diff
@@ -1,3 +1,8 @@
 # suppress inspection "UnusedProperty" for whole file
 application.namespace=file-transfer-service
-spring.servlet.multipart.max-file-size=${MAX_FILE_SIZE:300MB}
\ No newline at end of file
+spring.servlet.multipart.max-file-size=${MAX_FILE_SIZE:300MB}
+######### AWS S3 Credentials #########
+aws.secretAccessKey=${AWS_SECRET_ACCESS_KEY}
+aws.accessKeyId=${AWS_ACCESS_KEY_ID}
+aws.s3Path=${IMAGE_CLOUD_S3_PATH}
+aws.protocol=${S3_PROTOCOL}
```

Comment:
This wouldn't often need configured. Most of the time https will be the correct value. I think we should set it to http and allow over-riding through environment variables when a different value is required. This would remove the need to specify the protocol in most cases, but retain the ability to configure it.
```suggestion
# Can be over-ridden by setting the environment variable AWS_PROTOCOL. https://docs.spring.io/spring-boot/docs/2.1.8.RELEASE/reference/html/boot-features-external-config.html
aws.protocol=https
```